packages/snaps-rpc-methods/src/utils.ts

```diff
@@ -209,3 +211,30 @@ export async function getNode({
     ],
   });
 }
+
+/**
+ * Get the interface data from the associated RPC Method params.
+ *
+ * @param snapId - The Snap ID.
+ * @param params - The RPC Request params.
+ * @param getInterface - The Interface Controller function to get the interface.
+ * @param createInterface - The Interface Controller function to create the interface.
+ * @returns An object with the inteface ID and the interface content.
+ */
+export function getParamsInterface(
```

Comment:
Seems like this could be defined in the dialog RPC method instead? 🤔 

Or do you expect we need this elsewhere? I don't love passing the hooks around

Reply:
Yes, I just thought that we might have more RPC calls that returns UI in the future but I'll just put this logic in the RPC method directly for now